bgpd/bgp_mplsvpn.h

```diff
@@ -107,11 +107,8 @@ static inline int vpn_leak_to_vpn_active(struct bgp *bgp_vrf, afi_t afi,
 }
```

Comment:
Again, not modified by this commit, but isn't there a repetition in function vpn_leak_postchange() for the TOVPN case - vpn_leak_from_vrf_update_all() called twice?

Reply:
Removed duplicate call, not sure how that sneaked in there. Thanks!